### PR TITLE
Fix errcheck issues in server/channels/app/platform/busy_test.go

### PR DESCRIPTION
**Pull request by AulakhHarsh**
#### Summary
This PR fixes the errcheck issues reported by golangci-lint in the server/channels/app/platform/busy_test.go file.

#### Ticket Link

Fixes https://github.com/mattermost/mattermost/issues/29262
Jira https://mattermost.atlassian.net/browse/MM-61760

#### Release Note

```release-note

```

### CODE REVIEW

File: server/channels/app/platform/busy_test.go [hunk 1]
```diff
@@ -119,11 +119,15 @@ func compareBusyState(t *testing.T, busy1 *Busy, busy2 *Busy) bool {
 // ClusterMock simulates the busy state of a cluster.
 type ClusterMock struct {
 	Busy *Busy
+	t    *testing.T
```
Comment (hanzei) [inline server/channels/app/platform/busy_test.go:122]: You also need to assign the `t` when creating a `ClusterMock`

**Review by hanzei — CHANGES_REQUESTED** [review]
Thanks for the PR :+1:

File: server/channels/app/platform/busy_test.go (hunk 1 above)
Comment (AulakhHarsh) [reply server/channels/app/platform/busy_test.go:122]: Done

**Comment by hanzei** [comment]
Thanks @AulakhHarsh :+1:

**Pull request merged**